DATAREPO_CLIENT_README.md

```diff
@@ -0,0 +1,34 @@
+# Terra Data Repo Client project
+The tdrclient directory within the data repo code base contains an independent
+gradle project.
+
+## Build Targets
+The project has two targets that are interesting to use.
+
+### build
+The build target generates the OpenAPI client code for using the swagger source from the
+data repo code base. The code is generated into the `src/` directory. That allows the
+normal gradle java plugin to easily run the compile, build and test.
+
+### artifactoryPublish
+The artifactoryPublish target generates the extra files (pom.xml and so forth) needed to
+publish the client into the Broad Artifactory.
+
+In order to publish, you must know the artifactory username and password. You supply
+those to the gradle build by setting these environment variables:
+- ARTIFACTORY_USER
+- ARTIFACTORY_PASSWORD
```

Comment:
Where did you get these? Will we always use the same one, or everyone will need their own? If it's one username/password for everyone, I think it should be in Vault and we should pull it the same way we we do the PEM file.

Reply:
I made those envvars up. There is a single username/password for everyone that is shared. It is not a secure system.

This is the same model I use for Stairway publishing and similar to how other libraries are published. No one is using vault right now.

I'd be happier if it were vaulted and recovered from there. Is it OK with you to make that a separate ticket?

Reply:
Sounds good.